lib/route/nexthop.c

```diff
@@ -136,6 +136,14 @@ int rtnl_route_nh_compare(struct rtnl_nexthop *a, struct rtnl_nexthop *b,
 	return diff;
 }
 
+int rtnl_route_nh_identical(struct rtnl_nexthop *a, struct rtnl_nexthop *b)
```

Comment:
could you please add a short doc-text for public API?

Reply:
Sure thing, will add one